spec/support/javascript.rb

```diff
@@ -7,7 +7,9 @@ def fill_in_trix_editor(id, value)
   end
 
   def select_from_chosen(item_text, options)
-    option_value = page.evaluate_script("$(\"##{options[:from]} option:contains('#{item_text}')\").val()")
+    option_value = page
+                   .evaluate_script("$(\"##{options[:from]} "\
+                                      "option:contains('#{item_text}')\").val()")
```

Comment:
I think this version is harder to understand. I suggest an inline exception instead.